### ISSUE TITLE
Touch Bar action fires twice on mail.app

### ISSUE THREAD

**Issue by meidji** [folivoraAI/BetterTouchTool#2395]
**IMPORTANT:** Don't post private requests here (e.g. license issues, etc.). Please send me an email at boastr.net@gmail.com instead.

---

#### Description of bug/feature request/question:
Action from Touch Bar fires twice. 
I have configured App Specific action for Mail.app
Touch Bar Button name: Move to Archive
Predefined Action: Trigger Menubar Menu-Item
Path: Message;Move to;1;Work
(1 and Work - names of folders in my mail account)

Problem occurs after i have change laptop to new one.

#### Affected input device (e.g. MacBook Trackpad, Magic Mouse/Trackpad, Touch Bar, etc.):
Touch Bar

#### Device information:

Type of Mac: MacBook Pro (15-inch, 2017)
macOS version: 10.12.6 (16G1212)
BetterTouchTool version: 2.422 

#### Additional information (e.g. StackTraces, related issues, screenshots, workarounds, etc.):

I have latest version of BTT and have restarted my system! 

---

**Note:** Before bug reporting, please make sure you have the latest version of BetterTouchTool and that you have already tried to restart your system :-). If you encounter a crash, please attach a crash log from the macOS Console.app from the "User Diagnostic Reports" section.

**Comment by golodenko** [comment]
@fifafu is there a way to block repeating when I press a touchbar custom widget? 
I  run applescript and it runs like 3-4 times even with quick tap.

**Comment by meidji** [comment]
@fifafu i have sent mail with debug to you. Thanks for help, in advance!

**Comment by fifafu** [comment]
v2.430 alpha now has options to set repeatRate, repeatDelay and also the option to set a different action for "long presses"

**Comment by meidji** [comment]
Hey, @fifafu thanks for the patch! It is useful too. 
I figured out my case. 
If it will help someone, it would be great!

So, I had this path for menubar item: Message;Move to;1;Work
"message" and "Move to" - standard menu items,
"1" and "Work" - my mailboxes. "Work" is included in "1"
 
I don't know why, i decided that this is different layers and i need to use ";" (see attach).
That's why I had one email, that was moved to "1" mailbox, and second email, that was moved to "Work" mailbox. 

The issue was in path. Correct path for my case: Message;Move to;Work
So, there is no problem with additional action. 

Thanks! :)
![2018-06-07_20-02-52](https://user-images.githubusercontent.com/4648118/41115144-45d081b6-6a8f-11e8-9702-f727b38d2efb.png)
